<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompraAdjacentes</class>
 <widget class="QWidget" name="CompraAdjacentes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Fecha Emisión</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tipo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Serie</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Número</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Código</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tipo_ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Persona_ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_orden">
       <property name="text">
        <string>Orden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_guia">
       <property name="text">
        <string>Guia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_boleta">
       <property name="text">
        <string>Boleta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_factura">
       <property name="text">
        <string>Factura</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_flete">
       <property name="text">
        <string>Flete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ncredito">
       <property name="text">
        <string>N. Crédito</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ndebito">
       <property name="text">
        <string>N. Débito</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_modificar">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_eliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_salir">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_orden</tabstop>
  <tabstop>pushButton_guia</tabstop>
  <tabstop>pushButton_factura</tabstop>
  <tabstop>pushButton_flete</tabstop>
  <tabstop>pushButton_ncredito</tabstop>
  <tabstop>pushButton_ndebito</tabstop>
  <tabstop>pushButton_modificar</tabstop>
  <tabstop>pushButton_eliminar</tabstop>
  <tabstop>pushButton_salir</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
